stock_prices/stock_prices.py

```diff
@@ -1,15 +1,29 @@
-#!/usr/bin/python
+# !/usr/bin/python
 
-import argparse
+# import argparse
 
 def find_max_profit(prices):
-  pass
+  profits = []
```

Comment:
nice job! 

a way to solve this without 3 for loops would be like this

min_price = prices[0]
max_profit = prices[1] - min_price

for i in range(1, len(prices)):
____price = prices[i]
____max_profit = max(price - min_price, max_profit)
____min_price = min(price, min_price)

return max_profit